tensorboard/plugins/hparams/api_test.py

```diff
@@ -155,6 +155,63 @@ def test_summary_pb(self):
         expected_experiment_pb,
     )
 
+  def _assert_unique_summary(self, logdir, summary_pb):
+    """Test that `logdir` contains exactly one summary, `summary_pb`.
+
+    Specifically, `logdir` must be a directory containing exactly one
+    entry, which must be an events file of whose events exactly one is a
```

Comment:
Strictly speaking this doesn't verify that there aren't additional summaries after the first one.  The test passes immediately after finding the first valid summary.

It's a little less pretty but in the past I've just basically done

```python
events = list(tf.compat.v1.train.summary_iterator(events_file))
self.assertEqual(len(events), 2)
self.assertEqual(events[1].summary, summary_pb)
```